#06_Oleksandr Korniushyn/index.js

```diff
@@ -0,0 +1,93 @@
+// Написати цикли, які зможуть:
+
+// Вивести на сторінку в один рядок через кому числа від 10 до 20.
+// let i = 10;
+// do {
+//   document.write(` ${i}, `);
```

Comment:
Please use `console.log`, or `alert` next time. Construct a string, store it in a variable, and show it using mentioned methods.